eng/scripts/KillProcesses.ps1

```diff
@@ -1,16 +1,24 @@
 $ErrorActionPreference = 'Continue'
 
-taskkill /T /F /IM dotnet.exe
-taskkill /T /F /IM testhost.exe
-taskkill /T /F /IM iisexpress.exe
-taskkill /T /F /IM iisexpresstray.exe
-taskkill /T /F /IM w3wp.exe
-taskkill /T /F /IM msbuild.exe
-taskkill /T /F /IM vbcscompiler.exe
-taskkill /T /F /IM git.exe
-taskkill /T /F /IM vctip.exe
-taskkill /T /F /IM chrome.exe
-taskkill /T /F /IM h2spec.exe
+function _kill($processName) {
+    try {
+        & cmd /c "taskkill /T /F /IM $processName 2>&1"
+    } catch {
+        Write-Host "Failed to kill $processName: $_"
+    }
+}
+
+_kill dotnet.exe
+_kill testhost.exe
+_kill iisexpress.exe
+_kill iisexpresstray.exe
+_kill w3wp.exe
+_kill msbuild.exe
+_kill vbcscompiler.exe
+_kill git.exe
+_kill vctip.exe
+_kill chrome.exe
+_kill h2spec.exe
```

Comment:
What does this really change? The failure message `taskkill` spits out wasn't horrendous.

Reply:
The default view in AzDO shows all stderr messages (even though we are ignoring them). I'm trying to address this:

![image](https://user-images.githubusercontent.com/2696087/53187101-38d0b500-35b7-11e9-9ac2-a036735f340b.png)